toolkit/tools/go.mod

```diff
@@ -29,15 +29,26 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
```

Comment:
The changes to this `go.mod` file seem very odd. Typically, there are only 2 `require` sections: 1 for direct dependencies and 1 for indirect dependencies.

I recommend reverting the changes to this file and then running `go mod tidy` to pick up the new dependencies.

Reply:
I need to rebase as well, will clean it once.

Reply:
weird.. I had to remove the whole section for it to regenerate properly.